File: app/src/main/java/android/blessed/com/holefinder/network/ServerAPI.java
package android.blessed.com.holefinder.network;

import android.blessed.com.holefinder.models.Brand;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ServerAPI {
    @GET("brands")
    Call<List<Brand>> getBrands();
}

File: app/src/main/java/android/blessed/com/holefinder/network/ServerAPI.kt
package android.blessed.com.holefinder.network

import android.blessed.com.holefinder.models.RoadsResponse
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Url

interface ServerAPI {
    interface OSM {
        @GET
        fun getRoads(@Url url: String) : Observable<RoadsResponse>
    }
}